RecoLocalTracker/SiPixelRecHits/src/PixelCPEGeneric.cc

```diff
@@ -51,6 +51,25 @@ PixelCPEGeneric::PixelCPEGeneric(edm::ParameterSet const& conf,
   inflate_errors = conf.getParameter<bool>("inflate_errors");
   inflate_all_errors_no_trk_angle = conf.getParameter<bool>("inflate_all_errors_no_trk_angle");
 
+  // xtalk-related parameters
+  if (conf.exists("correct_xtalk")) {
```

Comment:
it would be better to group all the optional Phase 2 xtalk parameters into a single optional PSet. Right now, there is no guarantee that e.g. "induced_thre" is set if "correct_xtalk" is set.